docs/configuration/v1.x/metrics/azure-sql-database.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Azure SQL Database Declaration
+---
+
+## Azure SQL Database - ![Availability Badge](https://img.shields.io/badge/Available%20Starting-v1.0.0-green.svg)
+
+You can scrape an Azure SQL Database via the `AzureSqlDatabase` resource type.
+
+The following fields need to be provided:
+
+- `serverName` - The name of the SQL Server instance.
+- `databaseName` - The name of the database.
+
```

Comment:
Would you mind listing your custom metrics here please: https://promitor.io/configuration/v1.x/metrics/service-bus-queue

Reply:
Done.